components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/OAuthTokenReqMessageContext.java

```diff
@@ -42,7 +45,7 @@ public class OAuthTokenReqMessageContext {
 
     private long refreshTokenvalidityPeriod = OAuthConstants.UNASSIGNED_VALIDITY_PERIOD;
 
-    private long accessTokenIssuedTime;
+    private Timestamp accessTokenIssuedTimestamp;
```

Comment:
why we need this modification?

Reply:
This was done, so that we update the message context with the issued time and later we read from message context and set back to the token.
However it doesn't require to change this field to a Timestamp field. Will update with existing fields

Reply:
Reverted back still setting the issued time from message context